packages/nitro-protocol/contracts/ForceMove.sol

```diff
@@ -395,6 +395,7 @@ contract ForceMove is IForceMove {
     function _recoverSigner(bytes32 _d, Signature memory sig) internal pure returns (address) {
         bytes32 prefixedHash = keccak256(abi.encodePacked(prefix, _d));
         address a = ecrecover(prefixedHash, sig.v, sig.r, sig.s);
+        require(a != address(0), 'Invalid signature');
```

Comment:
Error message could specify `— null address is not a valid signer`?

Reply:
I took inspiration from here https://github.com/OpenZeppelin/openzeppelin-contracts/blob/ef32c04795e0288c0f71b08352b1877ddb509832/contracts/cryptography/ECDSA.sol#L60

I